go/checks/checks.go

```diff
@@ -254,6 +254,17 @@ func CheckMaxContributionsPerPartition(maxContributionsPerPartition int64) error
 	return nil
 }
 
+// CheckContributionBoundingOptions returns an error if exactly one of MaxContributions and MaxPartitionsContributed is not set.
+func CheckContributionBoundingOptions(maxContributions, maxPartitionsContributed int64) error {
```

Comment:
FYI: We might need to extend this in the future to also check for maxContributionsPerPartition, but not in this PR.